openfisca_core/tests/assets/variable_parameters.yaml

```diff
@@ -0,0 +1,99 @@
+# Use the VAR node and its condition children
+# to avoid using variations of parameter names to express conditions,
+# and compute the condition in your formula.
+
+# This is very useful for "terminal variations",
+# that is variations that happen close to the leaf nodes of the param tree
+
+# conditions use real variable names that you will pass from your formula
+# TODO or should we compute them for you ? Ideally yes, but what about period conversions ?
+
+# Regular value parameters :
+
+- variable: fillon_taux_max
+  description: Taux maximum de l'allègement général
+  format: percent
+  VAR:
+    - condition: "20 <= effectif_entreprise"
```

Comment:
👍 
This is way more explicit than having a `fillon_taux_max_moins_20_emloyes` and another `fillon_taux_max_moins_20_emloyes`.